pages/content-ui/package.json

```diff
@@ -10,15 +10,14 @@
   ],
   "scripts": {
     "clean": "rimraf ./dist",
-    "build:tailwindcss": "tailwindcss -i src/tailwind-input.css -o src/tailwind-output.css",
-    "build": "pnpm run clean && pnpm type-check && pnpm build:tailwindcss && vite build",
-    "build:watch": "cross-env __DEV__=true vite build --mode development & pnpm build:tailwindcss -- --watch",
+    "build:tailwindcss": "pnpm dlx tailwindcss -i ./src/tailwind-input.css -o ./dist/tailwind-output.css -m",
```

Comment:
<img width="850" alt="스크린샷 2024-08-14 오후 8 45 29" src="https://github.com/user-attachments/assets/765dd772-9d09-4d75-a2c5-1faf97c19c42">

I saw above error message. How about moving it inside `src`, the original file's location, instead of the `dist` folder?

Is there a reason you moved the output file to dist?

```suggestion
    "build:tailwindcss": "pnpm dlx tailwindcss -i ./src/tailwind-input.css -o ./src/tailwind-output.css -m",
```

Reply:
@Jonghakseo I want to separate output file from src, and there's entire /dist ignored.

Reply:
- https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/608

Like the PR above, It seems like a good idea to exclude from VSC for output files.

Reply:
@Jonghakseo What do you mean about 'out of vsc'?

Reply:
That bug occur probably because of the lack of '&'
In `build-watch` script:
`development & pnpm build:tailwindcss`

Not it's done by `concurrently`

And i check it a minute ago on VSC, and works perfect, `postinstall` isn't necessary.

Reply:
Sorry. It was too short. I mean Version Control System.

> @Jonghakseo What do you mean about 'out of vsc'?

Reply:
<img width="845" alt="스크린샷 2024-08-18 오후 2 18 38" src="https://github.com/user-attachments/assets/3bef6340-2509-46ab-b879-d9f5045d8210">

For me, I still see the above error message.
I just run `pnpm dev` after `pnpm clean` and I can see it right away.

Reply:
@Jonghakseo Yeah, that's reproducible, i need to fix it.

Reply:
@Jonghakseo Ok, I added a tw prebuild before, `postinstall` can't solve that issue, but this kind of prebuild did it.

Reply:
I'm gonna check it!

Reply:
HMR was not working. so i fixed it. 

[fix: use dev env](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/654/commits/3f3bd09ba9f3fd98537f325857fde3a8ac49a50b)